datasets/nrel-pds-nsrdb.yaml

```diff
@@ -37,6 +37,14 @@ Resources:
     ARN: arn:aws:s3:::nrel-pds-nsrdb/india/
     Region: us-west-2
     Type: S3 Bucket
+  - Description: Meteorological Statistical Model 1 (MTS1) data (1961-1990) in [HDF5](https://portal.hdfgroup.org/display/HDF5/HDF5) format
+    ARN: arn:aws:s3:::nrel-pds-nsrdb/mts1
+    Region: us-west-2
+    Type: S3 Bucket
+  - Description: Meteorological Statistical Model 2 (MTS2) data (1991-2005) in [HDF5](https://portal.hdfgroup.org/display/HDF5/HDF5) format
+    ARN: arn:aws:s3:::nrel-pds-nsrdb/mts1

```

Comment:
Should this be a different ARN?

Reply:
@jflasher MTS1 and MTS2 are NREL's irradiance models that pre-date the NSRDB. Do you still want us to put them in a different bucket?

Reply:
They're both using the mst1 prefix in the ARN, just wasn't sure if one should be `mst2`.

Reply:
Great catch, fixed, thanks!